src/index.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import Home from "./pages/home";
+// import Home from "./pages/home";
+import About from "./pages/about";
 import 'bootstrap/dist/css/bootstrap.min.css';
 import "./index.css"
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 
-ReactDOM.createRoot(document.getElementById('root')).render(<Home />);
+ReactDOM.createRoot(document.getElementById('root')).render(<About/>);
```

Comment:
Why is it changed to <About /> ??
It changes the root render to About Page. 

This page should be a different route rather than replacing the home route. 
Always build a habit of testing it on local before creating a PR. 

This makes the PR totally unacceptable. Not  looking further changes. :x: